tests/unit/schema/test.homepage_url.js

```diff
@@ -5,36 +5,34 @@ import { validateAddon } from 'schema/validator';
 import { validManifest } from './helpers';
 import { assertHasMatchingError } from '../helpers';
 
-describe('/homepage_url', () => {
-  const validURLs = [
-    'https://example.com/some/page',
-    'http://foo.com',
-    '__MSG_foo^&#__',
-  ];
+// eslint-disable-next-line jest/no-export
+export const VALID_URLS = [
+  'https://example.com/some/page',
+  'http://foo.com',
+  '__MSG_foo^&#__',
```

Comment:
do we have any test that is asserting that if the url is localized and the localized string is one of the restricted ones then we do mark the property as invalid as expected? (I haven't checked if it is abstracted out and not immediately visible)

Reply:
I don't see anything in https://github.com/mozilla/addons-linter/pull/925 or https://github.com/mozilla/addons-linter/pull/3415

Reply:
I don't think we actually support this case at all based on the existing code (in the main branch). I don't see a lot of code around localized strings either, though.

Reply:
yeah, I also just took a look and "preprocess" isn't considered at all at the moment and even besides that it doesn't seems we do have anything in the linter that is  taking care of preprocessing the manifest properties that can technically be localized (and then make sure the resulting manifest values are still valid).

Let's leave that as a separate follow up.

I looked into the existing and closed issues and I couldn't find any related to preprocess localized manifest properties to ensure the localized value is valid from a linter perspective, and so we should file a new one.

Reply:
> Let's leave that as a separate follow up.

+1

This reminds me of https://github.com/mozilla/addons-linter/issues/3911